packages/elpi/elpi.1.8.0/opam

```diff
@@ -0,0 +1,84 @@
+opam-version: "2.0"
+maintainer: "Enrico Tassi <enrico.tassi@inria.fr>"
+authors: [ "Claudio Sacerdoti Coen" "Enrico Tassi" ]
+license: "LGPL 2.1+"
+homepage: "https://github.com/LPCIC/elpi"
+doc: "https://LPCIC.github.io/elpi/"
+dev-repo: "git+https://github.com/LPCIC/elpi.git"
+bug-reports: "https://github.com/LPCIC/elpi/issues"
+
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  [make "tests"] {with-test & os != "macos"}
```

Comment:
I'm not too happy with this but I'm willing to let it pass if it's too annoying to do. The makefile target is calling `dune build` without either `-p` or `--profile` and is likely to break in the future since dune enables some warnings as errors by default. Would it be possible for you to either move the rules in dune and just call `dune runtest -p name -j jobs` or have settable variable in your makefile to trigger dev vs. release profile?

Reply:
Here the makefile snippet
```make
tests:
        dune build $(INSTALL)/bin/elpi
        dune build $(BUILD)/tests/test.exe
        ulimit -s $(STACK); \
                $(BUILD)/tests/test.exe \
                --seed $$RANDOM \
                --timeout $(TIMEOUT) \
                $(TIME) \
                --sources=$(shell pwd)/tests/sources/ \
                --plot=$(shell pwd)/tests/plot \
                $(addprefix --name-match ,$(ONLY)) \
                $(addprefix --runner , $(RUNNERS))
```
Where do you want me to pass `-p`?

I'd like to keep the target in make, since I need `ONLY` and `RUNNERS` to be overrideable via the command line (and computed via `wildcard` if not passed), as well as `TIME` & co.

But I'm happy to pass make something like `DUNE_OPTS` or `DUNE_PACKAGE` and use it where it fits.

Reply:
> Where do you want me to pass `-p`?

The two first lines only (the two calls to dune if I'm not mistaken)

Reply:
OK thanks for clarifying. I made the patch to Makefile as upstream, and added the specific commit to the opam package via `patches:`